test-support/helpers/x-select.js

```diff
@@ -9,25 +9,25 @@ import jQuery from 'jquery';
  * @param {string} texts - text of the option you are picking
  */
 export function select(selector, ...texts) {
-  let $select = selector instanceof jQuery ? selector : Ember.$(selector);
-  let $options = $select.find(`option`);
+  const $select = selector instanceof jQuery ? selector : Ember.$(selector);
+  const $options = $select.find(`option`);
 
   if (!$options.length) {
-    throw `No options found in ${selector}` ;
+    throw `No options found in ${selector}`;
   }
 
   $options.each(function() {
-    let $option = Ember.$(this);
+    const $option = Ember.$(this);
 
     Ember.run(() => {
-      this.selected = texts.some((text) => {
+      const selected = texts.some((text) => {
```

Comment:
This burned me in a test. For some reason, `texts.some(func)` would return false, but `this.selected` was still set to true. As I looked through the code, I saw no reason why selected was a class property.